### PR TITLE
Remove feature switch for secp256k1 program

### PR DESCRIPTION
**Pull request by jstarry**
#### Summary of Changes
Removed feature switch for secp256k1 program

### CODE REVIEW

**Comment by jstarry** [comment]
@ryoqun would you mind doing a sanity check of this?

File: ledger-tool/src/main.rs [hunk 1]
```diff
@@ -2386,13 +2386,13 @@ fn main() {
                             let mut store_failed_count = 0;
                             if force_enabled_count >= 1 {
                                 if base_bank
-                                    .get_account(&feature_set::secp256k1_program_enabled::id())
+                                    .get_account(&feature_set::spl_token_v2_multisig_fix::id())
```
Comment (ryoqun) [inline ledger-tool/src/main.rs:2389]: heh, nice shift of burden to more historic feature flag... ;)

**Comment by ryoqun** [comment]
I checked this pr doesn't introduce any compatibility concerns after reviving (yet again ^ 10) my live-cluster ci job: https://buildkite.com/solana-labs/solana/builds/50713#e0fa009b-430e-4d48-bdd6-c476ba877b1e

**Review by ryoqun — APPROVED** [review]
LGTM
nice cleaning!

**Pull request merged**